tests/ci_build/deploy_jvm_packages.sh

```diff
@@ -20,15 +19,7 @@ rm -rf ../build/
 
 # Re-build package without Mock Rabit
 # Deploy to S3 bucket xgboost-maven-repo
-if [[ "$build_gpu" == "0" ]]
-then
-  # Build CPU artifact
-  mvn --no-transfer-progress package deploy -P release-to-s3 -Dspark.version=${spark_version} -DskipTests
-else
-  # Build GPU artifact
-  sed -i -e 's/<artifactId>xgboost\(.*\)_\(.*\)<\/artifactId>/<artifactId>xgboost\1-gpu_\2<\/artifactId>/' $(find . -name pom.xml)
```

Comment:
This line was an ugly hack to create GPU-enabled artifact. Now that we have a proper `xgboost4j-gpu` module, this hack is not necessary any more.